### PR TITLE
Point 4

### PR DESCRIPTION
**Pull request by f00se**
Almost half of the job.

Admin can see a link "navigate to the admin panel" somewhere on the page. He can click the link to display that page (accessible only by admins).

### CODE REVIEW

**Comment by Exelord** [comment]
Do not spend a lot of time on styling. It's not required by me. It's like nice to have.

File: app/controllers/admin_controller.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  def index
+    show_welcome_once
+  end
+
+  private
+
+  def admin_permissions
+    authenticate_user!
+
+    redirect_to root_path, alert: t('admin.only_admins') unless
```
Comment (Exelord) [inline app/controllers/admin_controller.rb]: If you are using one line statements we prefer to write the in one line ;)
like: 
``` ruby
redirect_to root_path, alert: t('admin.only_admins') unless current_user.role? :admin
flash.now[:notice] = t('admin.admin_logged') if session[:first_visit_in_admin_panel].nil?
```

if the line is too long it means that refactor is required.

Comment (f00se) [inline app/controllers/admin_controller.rb]: Ok.

**Comment by Exelord** [comment]
Is it done already? :)

**Comment by f00se** [comment]
Yes.

**Pull request merged**